clients/admin-ui/src/features/privacy-experience/ConfigurePrivacyExperience.tsx

```diff
@@ -136,13 +138,36 @@ const ConfigurePrivacyExperience = ({
     TranslationWithLanguageName | undefined
   >(undefined);
 
-  const handleNewTranslationSelected = (translation: ExperienceTranslation) => {
+  const [translationIsOOB, setTranslationIsOOB] = useState<boolean>(false);
+
+  const getOOBTranslation = (language: SupportedLanguage) => {
+    const translation = passedInTranslations?.find(
+      (t) => t.language === language
+    );
+    if (translation) {
+      setTranslationIsOOB(true);
+    }
```

Comment:
nit: this "side effect" is surprising in a `get***` function, to me. If you inline this function into `handleCreateNewTranslation` like I suggest above, that'll solve this nit!